website/docs/reference/activation-strategies.md

```diff
@@ -161,12 +164,61 @@ If the constraint uses a standard Unleash context field, set the context field t
 
 Unleash SDKs expect all context values to be strings. If you use an operator that acts on non-string values, such as [numeric operators](#numeric-operators) or [date and time operators](#date-and-time-operators), the SDKs attempt to convert the string into the expected type. If the conversion fails, the constraint evaluates to `false`.
 
-## Custom activation strategies
 
-[Custom activation strategies](/reference/custom-activation-strategies)
+## Strategy types
+
+When you add an activation strategy to a flag, you can choose from several options, depending on how you want to target and roll out your feature.
+
+### Default strategy
+
+The default activation strategy is a type of gradual rollout strategy that you configure once per project and environment. It defines your most common configuration and is available as a quick option when adding strategies to flags.
+
+#### Configure the default strategy
+
+1. Go to the project where you want to configure the default strategy.  
+2. Open **Settings** and select **Default strategy**.  
+3. Choose the environment you want to edit.  
+4. Click **Edit default strategy for `<environment name>`**.  
+5. Configure your rollout, targeting, and variants, then click **Save strategy**.
+
+#### Add the default strategy to a flag
+
+To add your project’s default strategy, click **Add strategy** and select **Project default**.  
+In environments without an active strategy, you’ll also see a suggested option to apply your default strategy directly.
+
+### Standard strategies
+
+#### Gradual rollout
+
+Roll out a feature to a percentage of your users while ensuring each user has a consistent experience. You can combine gradual rollout with [targeting](#targeting) rules to support most rollout scenarios.
+
+#### Standard
+
+Turn the feature on or off for your entire user base. For more control, we recommend using a [gradual rollout](#gradual-rollout) strategy set to 100% instead.
+
+
+### Release templates
+
+[Release templates](/reference/release-templates) are a collection of activation strategies that let you reuse predefined rollout configurations across multiple flags. They make it easier to standardize rollout processes and reduce manual setup.
+
+### Advanced strategies
+
+#### IPs
+
+The `remoteAddress` strategy activates a feature flag for specific IP addresses.
+
+Parameters:
+- `IPs` – List of IPs to enable the feature for.
+
+#### Hosts
+
+The `applicationHostname` strategy activates a feature flag for specific hostnames.
+
+Parameters:
+- `hostNames` – List of hostnames to enable the feature for.
+
+### Custom activation strategies
+
+[Custom activation strategies](/reference/custom-activation-strategies) are deprecated and may be removed in a future major version. We recommend using the [gradual rollout](#gradual-rollout) strategy with constraints instead.
```

Comment:
We undeprecated these *sigh*. The recommendation is probably "don't use these unless you absolutely have to" right now

Reply:
Yes, they should be undeprecated. I'll make a PR to patch the UI. Thank you, very nice catch